src/rgw/rgw_rest_swift.cc

```diff
@@ -306,6 +306,8 @@ void RGWListBucket_ObjStore_SWIFT::send_response()
       s->formatter->dump_string("name", key.name);
       s->formatter->dump_string("hash", iter->meta.etag);
       s->formatter->dump_int("bytes", iter->meta.accounted_size);
+      if (iter->meta.user_data.size() && s->cct->_conf->rgw_swift_dlo_all_segments_size)
+        s->formatter->dump_string("total_manifest_size", iter->meta.user_data);
```

Comment:
@prallabh don't call this field `total_manifest_size`. This is a custom user data field, and it's application dependent. Maybe `user_custom_data`.